src/Symfony/Bundle/FrameworkBundle/DependencyInjection/Compiler/FormPass.php

```diff
@@ -53,11 +54,15 @@ public function process(ContainerBuilder $container)
         $typeExtensions = array();
 
         foreach ($container->findTaggedServiceIds('form.type_extension') as $serviceId => $tag) {
-            $alias = isset($tag[0]['alias'])
-                ? $tag[0]['alias']
-                : $serviceId;
+            $extendedType = $serviceId;
```

Comment:
I agree that it doesn't mean anything, but it's the default at the moment. Do you think we should require setting the `extended_type` attribute and trigger an exception if it is not set?